packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -281,9 +290,9 @@ def get_track_comments(args, track_id, current_user_id=None):
                     MutedUser.muted_user_id == Comment.user_id,
                     or_(
                         MutedUser.user_id == current_user_id,
+                        MutedUser.muted_user_id == current_user_id,
```

Comment:
we want to get muted user rows in the case where it's the current user who was muted. the logic before was kind of weird, it was depending on the join to exclude rows and then using the filter clause to confirm that?